purchase_request/tests/test_purchase_request_allocation.py

```diff
@@ -19,7 +19,7 @@ def setUp(self):
         self.product_product = self.env["product.product"].create(
             {
                 "name": "Product Product Test",
-                "type": "product",
+                "type": "consu",
```

Comment:
Should is_storable flag be set to True as well?

Reply:
adjusted